Tests/DatadogTests/Datadog/RUM/RUMFeatureTests.swift

```diff
@@ -108,59 +105,6 @@ class RUMFeatureTests: XCTestCase {
         XCTAssertEqual(request.allHTTPHeaderFields?["DD-REQUEST-ID"]?.matches(regex: .uuidRegex), true)
     }
 
-    // MARK: - HTTP Message with included variant
-
-    func testItUsesExpectedVariantQueryParaemterInMessage() throws {
-        let server = ServerMock(delivery: .success(response: .mockResponseWith(statusCode: 200)))
-        let httpClient = HTTPClient(session: server.getInterceptedURLSession())
-
-        let randomVariant: String = .mockAny()
```

Comment:
Moved to RUM's `RequestBuilderTests`. The only difference in this test and the next one is adding `variant` in URL query. Works much better if tested in pure unit-fashion.